### PR TITLE
added code for assignment1 CharactersEqualOccurrences

### PR DESCRIPTION
**Pull request by deepakghadge19**
Please review my PR for assignment1 CharactersEqualOccurrences

### CODE REVIEW

File: src/FirstAssignment/CharactersEqualOccurrences.java [hunk 1]
```diff
@@ -3,5 +3,33 @@
...
+		boolean b1;
+		b1 = areOccurrencesEqual("abacbc");
+		System.out.println("String abacbc is : "+b1);
+		b1 = areOccurrencesEqual("aaabb");
+		System.out.println("String aaabb is : "+b1);
+	}
+
+	public static boolean areOccurrencesEqual(String s) { 
+
+		int[] counts = new int[26];
```
Comment (amitsahu8) [inline src/FirstAssignment/CharactersEqualOccurrences.java:16]: format the code

File: src/FirstAssignment/CharactersEqualOccurrences.java [hunk 2]
```diff
@@ -3,5 +3,33 @@
...
+		int max=0;
+
+		for(char c : s.toCharArray())
```
Comment (amitsahu8) [inline src/FirstAssignment/CharactersEqualOccurrences.java:19]: Need to discuss the logic

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

File: src/FirstAssignment/CharactersEqualOccurrences.java (hunk 1 above)
Comment (deepakghadge19) [reply src/FirstAssignment/CharactersEqualOccurrences.java:14]: Added formatting in the code and Pushed code to same branch

File: src/FirstAssignment/CharactersEqualOccurrences.java [hunk 3]
```diff
@@ -1,7 +1,29 @@
 package FirstAssignment;
 
-
 public class CharactersEqualOccurrences {
 
+	public static void main(String[] args) {
+		boolean b1;
+		b1 = areOccurrencesEqual("abacbc");
```
Comment (amitsahu8) [inline src/FirstAssignment/CharactersEqualOccurrences.java:7]: no need of b1 and if you need to use it, provide the meaningful name

File: src/FirstAssignment/CharactersEqualOccurrences.java [hunk 4]
```diff
@@ -1,7 +1,29 @@
...
+		System.out.println("String abacbc is : " + b1);
+		b1 = areOccurrencesEqual("aaabb");
+		System.out.println("String aaabb is : " + b1);
+	}
+
+	public static boolean areOccurrencesEqual(String s) {
+		int[] counts = new int[26];
```
Comment (amitsahu8) [inline src/FirstAssignment/CharactersEqualOccurrences.java:14]: array size should be dynamic

File: src/FirstAssignment/CharactersEqualOccurrences.java (hunk 3 above)
Comment (deepakghadge19) [reply src/FirstAssignment/CharactersEqualOccurrences.java]: Removed b1 from code

File: src/FirstAssignment/CharactersEqualOccurrences.java (hunk 4 above)
Comment (deepakghadge19) [reply src/FirstAssignment/CharactersEqualOccurrences.java]: Done

**Review by rohitlamsonge66 — APPROVED** [review]

**Pull request merged**